polling_stations/apps/data_collection/management/commands/import_central_bedfordshire.py

```diff
@@ -29,9 +31,17 @@ def address_record_to_dict(self, record):
         if uprn.endswith("0000000"):
             rec["uprn"] = ""
```

Comment:
I'm just going to merge this one, but if you spot any more where the UPRN data is ruined from being imported into Excel, can we bounce them back to the council if we're spotting them with weeks to go until polling day. Having the UPRNs makes the checking so much easier and they've got plenty of time to get back to us with a fixed file at this point in time.